src/tests/integration_test_utils.rs

```diff
@@ -119,23 +126,47 @@ pub fn lines_match(expected: &str, have: &str, skip: Option<usize>) {
         assert_eq!(
             &expected[indentation..],
             it.next().unwrap(),
-            "on line {} of input",
-            i + 1
+            "on line {} of input:\n{}",
+            i + 1,
+            delineated_string(have),
         );
     }
-    assert_eq!(it.next(), None, "more input than expected");
+    if !allow_partial {
+        assert_eq!(it.next(), None, "more input than expected");
+    }
+}
+
+pub fn lines_match(expected: &str, have: &str) {
+    lines_match_skip(0, expected, have)
+}
+
+pub fn delineated_string(txt: &str) -> String {
+    let top = "▼".repeat(100);
+    let btm = "▲".repeat(100);
+    let nl = "\n";
+    top + &nl + txt + &nl + &btm
 }
 
 pub struct DeltaTest {
     config: config::Config,
     calling_process: Option<String>,
+    explain_ansi_: bool,
 }
 
 impl DeltaTest {
     pub fn with(args: &[&str]) -> Self {
```

Comment:
I feel like with the renaming and refactoring it would now make sense to call this `with_args()`.

Reply:
OK -- that's nice and clear.